internal/messenger/email/email.go

```diff
@@ -30,7 +30,7 @@ type Server struct {
 
 	// Rest of the options are embedded directly from the smtppool lib.
 	// The JSON tag is for config unmarshal to work.
-	smtppool.Opt `json:",squash"`
```

Comment:
This is meant to be `,squash` Ref: https://pkg.go.dev/github.com/mitchellh/mapstructure#hdr-Embedded_Structs_and_Squashing

Reply:
my bad. I'm really sorry about this. I would like to correct this mistake and raise the PR again. Can I?

Reply:
You can just commit a fix and amend this PR.

Reply:
Thanks for the guidance! I'm looking forward to contributing to this project.